.github/codecov.yml

```diff
@@ -0,0 +1,28 @@
+codecov:
+  # strict_yaml_branch: master # Tell Codecov to only look at the master branch version of the codecov.yml file
```

Comment:
We should run covecov on each branch.
Do we need a specific codecov yaml file per branch?
If yes, what would be different?

Reply:
On one hand I don’t think we need different versions on different branches, but if we get a low coverage on a version branch it might be useful to tune the parameters for it.
On the other hand I don’t think we must restrict Codecov to use the master branch version only. After the CP most future PRs will get this version anyway